Source/ReadModelRepositoryResourceTypeRepresentation.cs

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ * --------------------------------------------------------------------------------------------*/
+using System;
+using System.Collections.Generic;
+using Dolittle.Resources;
+
+namespace Dolittle.ReadModels.MongoDB
+{
+    /// <inheritdoc/>
+    public class ReadModelRepositoryResourceTypeRepresentation : IRepresentAResourceType
+    {
+        IDictionary<Type, Type> _bindings;
+        
+        /// <inheritdoc/>
+        public ResourceType Type => "readModels";
+        /// <inheritdoc/>
+        public ResourceTypeImplementation ImplementationName => "MongoDB";
+        /// <inheritdoc/>
+        public Type ConfigurationObjectType => typeof(ReadModelRepositoryConfiguration);
+        /// <inheritdoc/>
+        public IDictionary<Type, Type> Bindings {
```

Comment:
Simplification of the bindings - no need for `null`check nor explicit Initialize - use Dictionary initializers.
```suggestion
public IDictionary<Type, Type> Bindings => new Dictionary<Type,Type> {
   {typeof(IReadModelRepositoryFor<>), typeof(ReadModelRepositoryFor<>)}  
}
```